keanu-python/keanu/algorithm/sampling.py

```diff
@@ -20,3 +20,17 @@ def sample(net, sample_from, algo='metropolis', draws=500, drop=0, down_sample_i
     vertex_samples = {Vertex._get_python_id(vertex_unwrapped): list(map(Tensor._to_ndarray, network_samples.get(vertex_unwrapped).asList())) for vertex_unwrapped in vertices_unwrapped}
 
     return vertex_samples
+
+def iterate_samples(net, sample_from, algo='metropolis', draws=500, drop=0, down_sample_interval=1):
+	vertices_unwrapped = k.to_java_object_list(sample_from)
+
+	network_samples = algorithms[algo].withDefaultConfig().generatePosteriorSamples(net.unwrap(), vertices_unwrapped).dropCount(drop).downSampleInterval(down_sample_interval).stream().iterator()
+	
+	return iterator_samples(network_samples, draws, vertices_unwrapped)
+
+
+def iterator_samples(network_samples, draws, vertices_unwrapped):
+	for i in range(draws):
```

Comment:
Given you don't actually need the result you can just do:
for _ in range(draws):

This will just tell Python you're not bothered about the return value.

Reply:
nice!!! thanks